text/tests/unittests/automm/test_data_process_image.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import PIL
+from autogluon.text.automm.data.process_image import ImageProcessor
+
+from datasets import (
+    PetFinderDataset,
+    HatefulMeMesDataset,
+    AEDataset,
+)
+
+ALL_DATASETS = {
+    "petfinder": PetFinderDataset,
+    "hateful_memes": HatefulMeMesDataset,
+    "ae": AEDataset,
+}
+
+@pytest.mark.parametrize(
+    "augmentations",
+    [
+        {
+            "model.timm_image.train_transform_types": ["resize_to_square","center_crop"],
+            "model.timm_image.val_transform_types": ["resize_shorter_side","center_crop"]
+        },
+
+        {
+            "model.timm_image.train_transform_types": ["resize_shorter_side","center_crop"],
+            "model.timm_image.val_transform_types": ["resize_shorter_side","center_crop"]
+        },
+
+        {
+            "model.timm_image.train_transform_types": ["resize_shorter_side","center_crop","horizontal_flip","vertical_flip"],
+            "model.timm_image.val_transform_types": ["resize_shorter_side","center_crop"]
+        },
+
+        {
+            "model.timm_image.train_transform_types": ["resize_shorter_side","center_crop","affine","color_jitter","randaug"],
+            "model.timm_image.val_transform_types": ["resize_shorter_side","center_crop"]
+        },
+
+        {
+            "model.timm_image.train_transform_types": ["resize_shorter_side","center_crop","affine(15, (0.1, 0.1), (0.9, 1.1))"],
```

Comment:
The hyper-parameters' meanings are unclear if using tuple. I think we need to support dictionary, which is more informative. For example, we can do ast.literal_eval("{'degrees': 15, 'translate': (0.1, 0.1), 'scale': (0.9, 1.1)}").